pkgs/top-level/all-packages.nix

```diff
@@ -27989,6 +27989,16 @@ with pkgs;
 
   prusa-slicer = callPackage ../applications/misc/prusa-slicer { };
 
+  snapmaker-luban = callPackage ../applications/misc/snapmaker-luban {
+    branch = "stable";
+    inherit pkgs;
+  };
+
+  snapmaker-luban-latest = callPackage ../applications/misc/snapmaker-luban {
```

Comment:
nixpkgs generally doesn't package beta software unless the original release is really old. Also when it's beta it must be tagged as either unstable or beta, not latest

Reply:
Removed the beta.